Dockerfiles/agent/cont-init.d/50-ecs.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
-
-if [[ -z "${ECS_FARGATE}" ]]; then
+ 
+if [[ "${ECS_FARGATE}" == "true" ]]; then
```

Comment:
We would need to update https://github.com/DataDog/datadog-agent/blob/c5717efe0244d7fc8ebbd2ab5ee93f07e6c3fee8/pkg/config/env/environment.go#L65 as well to keep the same logic everywhere.